cst438_gradebook/src/main/resources/application.properties

```diff
@@ -3,14 +3,14 @@ server.servlet.session.persistent=false
 
 # registration service
 registration.url = http://localhost:8080
-registration.service = default
+registration.service = REST
```

Comment:
nit: not sure we're supposed to commit those values, since these change depending on testing.

Reply:
Yeah, I think I did a commit and push from Spring boot which didn't allow me to exclude this file